timeplus/client.go

```diff
@@ -276,7 +267,7 @@ func readCompleteLine(reader *bufio.Reader) (string, error) {
 	return string(line), nil
 }
 
-func (s *TimeplusClient) queryStreamV2(sql string, batchCount int, batchBufferTime int) (rxgo.Observable, func(), error) {
+func (s *TimeplusClient) queryStreamV2(sql string, batchCount int, batchBufferTime int) (*QueryInfo, rxgo.Observable, func(), error) {
```

Comment:
This is not a good API design, it returns too many values. We should wrap them into a `QueryResult` object instead.